include/mbedtls/sha512.h

```diff
@@ -182,12 +198,13 @@ MBEDTLS_DEPRECATED static inline void mbedtls_sha512_finish(
 }
 
 /**
- * \brief          SHA-512 process data block (internal use only)
+ * \brief          This function processes a single data block within
```

Comment:
Blocker: this function is for internal use only. Please keep this explicit.

Reply:
fixed